sympy/matrices/matrices.py

```diff
@@ -2924,6 +2927,9 @@ def berkowitz(self):
         berkowitz_eigenvals
         """
         from sympy.matrices import zeros
+        berk = [(1,)]
```

Comment:
This could be defined as a tuple:
``    berk = ((1,),) ``
(and returned as such below).